### ISSUE TITLE
ERROR --> Fatal: Tensor **** of type UINT8 is not supported

### ISSUE THREAD

**Issue by easy-and-simple** [kendryte/nncase#132]
Hello Guys,
I tried to convert model from tensorflow lite object detection example and got this error:
Fatal: Tensor (FeatureExtractor/MobilenetV1/MobilenetV1/Conv2d_0/weights_quant/FakeQuantWithMinMaxVars) of type UINT8 is not supported
What this means and what I need to do to fix it and to make this model compatible with nncase?

Model from yolo example included in nncase looks similar in extension and size but it is processed without issues. What is difference between these models?

**Comment by sunnycase** [comment]
You should use float model.

**Comment by easy-and-simple** [comment]
And why you closed my case if it is not resolved?

**Comment by easy-and-simple** [comment]
I have ready model that I want to use. If it is using uint8 then I need to covert it somehow=
nncase is never works and all the time gives error whatever I try to convert
I want to covert my model to kmodel3 but nncase 0.1 is not working and gives error that padding doesn't match and gives error
**Fatal: Layer Pad is not supported: Only paddings of [[0,0],[1,1],[1,1],[0,0]] is supported**
It always return this error no matter what padding I set.
Maybe issue is with conversion of my model from keras to tflite because there you don't supply converter and I am using tensorflow converter that probably is no supported by your nncase as ver 0.1 as well ver 0.2 because bot cant convert my tflite file. nncase 0.2 throw error Fatal: KPU allocator cannot allocate more memory.
What converter I need to use to convert h5 keras file to tflite file that support your nncase 0.1 or 0.2?
